### ISSUE TITLE
Conflicts with Tinkers' Construct 2

### ISSUE THREAD

**Issue by addtioe** [LakMoore/InfiniTiC#36]
[crash-2020-10-31_21.49.29-client.txt](https://github.com/LakMoore/InfiniTiC/files/5470089/crash-2020-10-31_21.49.29-client.txt)

**Comment by AnrDaemon** [comment]
No, that is because you registered the name that is owned by another mod.
Either don't do that, or load your mod last (move .jar it to subdir or rename to sort last in the list).
